src/point_one/fusion_engine/messages/control.h

```diff
@@ -383,6 +383,12 @@ enum class DeviceType : uint8_t {
   LG69T_AP = 3,
   /** Device is a Quectel LG69T-AH system. */
   LG69T_AH = 4,
+  /** Device is a Nexar Beam2K system. */
+  NEXAR_BEAM2K = 5,
+  /** Device is an SSR system on a LG69T. */
```

Comment:
"Point One SSR client running on an LG69T platform."

I don't think we need "Device is a" for all of these